selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/miscellaneous/FindUsagesBaseOperationTest.java

```diff
@@ -142,6 +142,7 @@ public void checkFindUsagesBaseOperation() {
     findUsages.selectNodeInFindUsagesPanel(PROJECT_NAME);
     findUsages.sendCommandByKeyboardInFindUsagesPanel(ENTER.toString());
     findUsages.waitExpectedTextIsNotPresentInFindUsagesPanel(EXPECTED_TEXT_1);
+    findUsages.selectNodeInFindUsagesPanel(PROJECT_NAME);
```

Comment:
Looks like there is separate step started from `findUsages.selectNodeInFindUsagesPanel(PROJECT_NAME);`.
It's not quite clear what second call is for. So, it would be better to divide two steps by empty row and comment on it.